backend/src/recommitment/recommitment.service.ts

```diff
@@ -270,6 +284,11 @@ export class RecommitmentService {
     templateRecipient: TemplateType;
   } {
     switch (status) {
+      case RecommitmentStatus.PENDING:
+        return {
```

Comment:
Just want to double check that the member recommitment status is updated as PENDING in recommitment table prior to calling this?